src/Resources/TortoiseProcENG.rc

```diff
@@ -690,39 +690,43 @@ BEGIN
                     "Button",BS_AUTOCHECKBOX | WS_TABSTOP,12,220,100,10
 END
 
-IDD_SETTINGSDIALOGS DIALOGEX 0, 0, 301, 231
+IDD_SETTINGSDIALOGS DIALOGEX 0, 0, 301, 254
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "General::Dialogs 1"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    GROUPBOX        "Log messages",IDC_STATIC,7,7,287,217
-    LTEXT           "&Font for log messages:",IDC_STATIC,14,17,92,13,SS_CENTERIMAGE
-    COMBOBOX        IDC_FONTNAMES,107,17,136,90,CBS_DROPDOWNLIST | CBS_OWNERDRAWVARIABLE | CBS_SORT | CBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP
-    COMBOBOX        IDC_FONTSIZES,253,17,31,47,CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    GROUPBOX        "Log messages",IDC_STATIC,7,7,287,240
+    LTEXT           "Default limitation of log messages:",IDC_STATIC,14,20,150,8,SS_CENTERIMAGE

```

Comment:
Adjust the settings optics:
1) Allow user to choose: No limit, remeber last selected date limit, used custom limit and then on the next line the user can specify it.